test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -624,15 +625,17 @@ def forward(self, x):
     def test_prepare_custom_config_dict_non_traceable_module_class(self):
```

Comment:
nit: wondering if we want to break the test into smaller ones so that it's easier to read

Reply:
and the first test can be simpler